### ISSUE TITLE
Crossbuild docker image for different architectures

### ISSUE THREAD

**Issue by Pentusha** [TwinProduction/gatus#54]
I'd like to run gatus on my raspberry pi cluster, but the docker image `twinproduction/gatus` is built only with `linux/amd64` architecture support.
PRI4 Output:
```
arch; docker run twinproduction/gatus
armv7l
standard_init_linux.go:211: exec user process caused "exec format error"
failed to resize tty, using default size
```

This is how you can make sure that the manifest does not contain any CPU architecture description.
```
docker buildx imagetools inspect twinproduction/gatus
```

I added few build steps to Github Actions for cross compilation of docker images and now gatus works on raspberry.

```
arch; docker run pentusha/gatus      
armv7l
2020/12/02 09:21:50 [config][Load] Reading configuration from configFile=config/config.yaml
2020/12/02 09:21:50 [config][validateAlertingConfig] Alerting is not configured
2020/12/02 09:21:50 [config][validateServicesConfig] Validated 5 services
2020/12/02 09:21:50 [main][main] Listening on 0.0.0.0:8080/
2020/12/02 09:21:51 [watchdog][monitor] Monitored serviceName=frontend; success=true; errors=0; requestDuration=648ms; 
2020/12/02 09:21:51 [watchdog][monitor] Monitored serviceName=backend; success=true; errors=0; requestDuration=218ms; 
2020/12/02 09:21:52 [watchdog][monitor] Monitored serviceName=monitoring; success=true; errors=0; requestDuration=223ms; 
```
```
docker buildx imagetools inspect pentusha/gatus | grep Platform
  Platform:  linux/amd64
  Platform:  linux/arm/v6
  Platform:  linux/arm/v7
  Platform:  linux/arm64
```
I also tried to some other platforms, but build was [failed](https://github.com/Pentusha/gatus/runs/1485076752?check_suite_focus=true) so I keep only ARM.

**Comment by TwinProduction** [comment]
Closed by #55

**Comment by TwinProduction** [comment]
Released in [v1.9.1](https://github.com/TwinProduction/gatus/releases/tag/v1.9.1)

P.S. I've also removed `linux/arm/v6`, as `linux/arm/v7` and `linux/arm64` covers all requested use cases right now. 
This also lowers the time for the action to complete, but I might re-add it if it becomes necessary in the future.
